pkg/kubelet/gpu/nvidia/nvidia_gpu_manager.go

```diff
@@ -101,8 +102,7 @@ func (ngm *nvidiaGPUManager) Start() error {
 	if err := ngm.discoverGPUs(); err != nil {
 		return err
 	}
-	// It's possible that the runtime isn't available now.
-	ngm.allocated = ngm.gpusInUse()
+
```

Comment:
I'm -1 on this. If containers recover and continue using GPUs, then allocatable will be wrong for the node.